tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -3494,6 +3488,38 @@ def clone_for_deployment(self,
         predictor_clone.save_space()
         return predictor_clone if return_clone else predictor_clone.path
 
+    @staticmethod
+    def _check_if_hyperparameters_handle_text(hyperparameters: dict) -> bool:
+        """Check if hyperparameters contain a model that supports raw text features as input"""
+        models_in_hyperparameters = set()
+        is_advanced_hyperparameter_type = False
+        for key in hyperparameters:
+            if isinstance(key, int) or key == 'default':
+                is_advanced_hyperparameter_type = True
```

Comment:
add `break` to skip the rest of the cycle?

Reply:
added